src/tracks/playabletrack/notetrack/ui/NoteTrackAffordanceControls.cpp

```diff
@@ -109,6 +110,9 @@ void NoteTrackAffordanceControls::Draw(TrackPanelDrawingContext& context, const
                 py >= clipRect.GetTop() && py <= clipRect.GetBottom());
 
         context.dc.SetClippingRegion(rect);
+        context.dc.SetTextBackground(wxTransparentColor);
+        context.dc.SetTextForeground(theTheme.Colour(clrClipNameText));
+        context.dc.SetFont(wxFont(wxFontInfo()));
```

Comment:
Only changing note track affordance text colors?

Reply:
In new commit history this commit contains also wave track colors changes